lib/sequelize.js

```diff
@@ -667,22 +632,16 @@ module.exports = (function() {
    *
    * @see {Model#build} for more information about callee.
    */
-  Sequelize.prototype.query = function(sql, callee, options, replacements) {
+  Sequelize.prototype.query = function(sql, callee, options) {
```

Comment:
Not sure, already buried in a shit tonne of work getting what i've removed so far to work, 5 hours so far and not green.